code/style.css

```diff
@@ -1,4 +1,150 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing:border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  background-color: whitesmoke;
 }
+
+.header {
+  width: 100%;
+  height: 100px;
+  background-color: whitesmoke;
+  padding: 30px 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 40px;
+}
+
+.logo {
+  line-height: 0.7;
+}
+
+.logo h1 {
+  font-size: 45px;
+}
+
+.news {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 40px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+p {
+  margin: 0 10px;
+  font-size: 14px;
+}
+
+h2 {
+  margin: 0 10px;
+  font-size: 27px;
+}
+
+h3 {
+  margin: 0 10px;
+  font-size: 20px;
+}
+
+.headnews img {
+  width: 100%;
+  height: auto;
+}
+
+.news2 img {
+  width: 100%;
+  height: auto;
+}
+
+.news3 img {
+  width: 100%;
+  height: auto;
+}
+
+.news4 img {
+  width: 100%;
+  height: auto;
+}
+
+.news5 img {
+  width: 100%;
+  height: auto;
+}
+
+
+/*for mobile*/
+@media (max-width: 767px) {
```

Comment:
Wondering here if you actually need a media query for mobile or if this section can just get into the section above (since mobile is your default)?